vendor/licenses/git_submodule/urscript-extension.dep.yml

```diff
@@ -1,22 +1,26 @@
 ---
-name: typst-grammar
-version: 180d59e6f4e09941d778b5fa6fc48d52211294a1
+name: urscript-extension
+version: 51d5c4021fccc1fd9549c01f80156d3e001ee5ed
```

Comment:
Why are you renaming and updating this file? The `script/add-grammar` script you should have run to add the grammar should have created a new file for the new grammar.

Please revert this change and add the correct license file.

Reply:
Ooof. I know why this happened. https://github.com/github-linguist/linguist/issues/7145 You'll need to redo this one that issue has been resolved.